fleet-charts/gemma-server/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
could we open source this entire repo with it?

Reply:
Hi @corentone - The repo already has the same LICENSE in the root: https://github.com/GoogleCloudPlatform/gke-fleet-management/blob/main/LICENSE

Best Practice is to (also in this case) include the Helm Charts' LICENSE in the Helm chart, as the charts are most often "pulled" directly and do not include the rest of the repo.